src/main/core/manager.c

```diff
@@ -593,11 +593,11 @@ DNS* manager_getDNS(Manager* manager) {
 }
 
 guint32 manager_getNodeBandwidthUp(Manager* manager, in_addr_t ip) {
-    return controller_getBandwidthUpBytes(manager->controller, ip) / 1024;
+    return controller_getBandwidthUpBytes(manager->controller, ip) / 1024 * 1024 / 1000 * 1000 / 1024;
```

Comment:
A comment here and below that this is to preserve legacy behavior, with a TODO to remove, would be useful.

Assuming you want to keep this "identical behavior" commit in the history, I'd lean towards adding another commit reverting it in this same PR, and then merging without squashing to preserve that history. This avoids having this weird state in the "main history" (git log --first-parent main), and the possibility of forgetting to revert it later.

Reply:
Added some comments and a commit to undo this behaviour.